src/pipecat/frames/frames.py

```diff
@@ -393,6 +404,25 @@ class StopInterruptionFrame(SystemFrame):
     pass
 
 
+@dataclass
+class UserStartedSpeakingFrame(SystemFrame):
+    """Emitted by VAD to indicate that a user has started speaking. This can be
+    used for interruptions or other times when detecting that someone is
+    speaking is more important than knowing what they're saying (as you will
+    with a TranscriptionFrame)
+
+    """
+
+    pass
+
+
+@dataclass
+class UserStoppedSpeakingFrame(SystemFrame):
```

Comment:
This was a `ControlFrame` which was actually not correct because it would be queued and we actually want to know the user stopped speaking before.